### PR TITLE
Describe development HEAD release process

### LINKED ISSUE #1114: Setup CI for nightly build
Setup a nightly build of a new docker image using CI.

### PR DESCRIPTION
**Pull request by bryk**
@hwangjr @colemickens

Fixes #1114

### CODE REVIEW

**Comment by bryk** [comment]
@rf232 Can you review this?

File: README.md [hunk 1]
```diff
@@ -20,14 +20,19 @@ If it is missing, you can install the latest stable release by running the follo
 ```bash
 kubectl create -f https://rawgit.com/kubernetes/dashboard/master/src/deploy/kubernetes-dashboard.yaml
 ```
-Note that for the metrics and graphs to be available you need to have [Heapster](https://github.com/kubernetes/heapster/) running in your cluster.
 
-## Usage
-You may access the UI via the apiserver proxy. Open a browser and navigate to `https://<kubernetes-master>/ui`.
+You can also install unstable HEAD release with newest features that the team works on by
```
Comment (rf232) [inline README.md]: not really a release is it, I think I'd call it unstable HEAD builds

Comment (rf232) [inline README.md]: > with newest features

with the newest features?

Comment (bryk) [inline README.md]: All makes sense, done.

**Pull request merged**